### DISCUSSION TITLE
bx-loading not spinning

### DISCUSSION THREAD

**Discussion by tpluscode** [carbon-design-system/carbon-web-components#678] [Q&A]
I'm starting with Carbon Web Components and apparently I missed some crucial step preparing my app.

I simply add a `<bx-loading>` to a blank page but it does not spin and takes 100% of the available width.

https://hnn63.sse.codesandbox.io/

What am I missing?

**Comment by asudoh** [comment]
Ref: #645

**Reply by tpluscode** [reply]
Oh 😊 

I totally agree with the comment that a spinner would ideally work out of the box :)
